manifests/mlops-sagemaker/networking-modules.yaml

```diff
@@ -0,0 +1,8 @@
+name: networking
+path: modules/optionals/networking/
+parameters:
+  - name: internet-accessible
+    value: True
+  - name: interface-endpoints
```

Comment:
No such parameter exists on the networking module....is this correct?

Reply:
I will remove it. But this could be a feature for future PR. I have modified the networking module to allow interface deployment of more interface endpoints. This is especially important if a lot of data is downloaded from S3 to SageMaker studio

Reply:
You can issue a FR for such a change....but this param does nothing but confuse users.  The modification you made to the networking module is not in this PR.  Please confirm that this PR does not need this enhancement.

Reply:
Ill raise a follow up PR for this. This does not need to be part of this PR and I will remove the change